Makefile

```diff
@@ -474,6 +474,28 @@ gen_genesis_and_config:
 clear_genesis_and_config:
 	rm build/config/gen.*.json
 
+.PHONY: localnet_up
+## Starts localnet with all dependencies (basically, `tilt up`)
```

Comment:
Good idea, but it doesn't seem `tilt` is meant to be installed via `go install`.
e.g. `go install "github.com/tilt-dev/tilt/cmd/tilt@v0.30.13"` fails, and it is not a supported way to install tilt. This would be convinient though..

Reply:
It is not supported indeed. https://github.com/tilt-dev/tilt/issues/5334#issuecomment-1121173647

Reply:
Why not just put `curl -fsSL https://raw.githubusercontent.com/tilt-dev/tilt/master/scripts/install.sh | bash` in the `install_cli_deps` target? It's call `cli_deps`, not `go_cli_deps` :)

Reply:
@Olshansk I guess you're right we can do that, though, I've been avoiding `curl *.sh | bash` due to security concerns. Okay. Let do this.